src/main/java/org/myrobotlab/opencv/OpenCVFilterSolr.java

```diff
@@ -58,12 +58,11 @@
   private static final long serialVersionUID = 1L;
   transient public final static Logger log = LoggerFactory.getLogger(OpenCVFilterSolr.class);
 
-  private CvFont font = cvFont(CV_FONT_HERSHEY_PLAIN);
-  private CvFont fontWarning = cvFont(CV_FONT_HERSHEY_PLAIN);
+  transient private CvFont font = cvFont(CV_FONT_HERSHEY_PLAIN);
 
   private String formattedSearchResult = "No Result";
 
-  private Solr solr = null;
+  transient private Solr solr = null;
   private String solrUrl = "http://localhost:8983/solr/wikipedia";

```

Comment:
good catch on the new transients,
I would like to see publishing or sending msgs
e.g. send(solar, 'doSomethingSolr', params) vs a direct reference
solar in this case being a String fieldname

Reply:
future pr.. this pr was more about making sure all the filters are serializable and to fix places where the frame converters don't behave as expected.